pkgs/development/tools/build-managers/waf/setup-hook.sh

```diff
@@ -1,20 +1,40 @@
 wafConfigurePhase() {
     runHook preConfigure
 
-    if ! [ -f ./waf ]; then
-        cp @waf@ waf
+    if ! [ -f "${wafPath:=./waf}" ]; then
+        echo "copying waf to $wafPath..."
+        cp @waf@ "$wafPath"
     fi
 
     if [[ -z "${dontAddPrefix:-}" && -n "$prefix" ]]; then
         configureFlags="${prefixKey:---prefix=}$prefix $configureFlags"
     fi
 
-    local flagsArray=(
-        $configureFlags ${configureFlagsArray[@]}
+    local flagsArray=(@crossFlags@);
+    for flag in $configureFlags "${configureFlagsArray[@]}";
```

Comment:
Some more flags we might want to add are ``` --bundled-libraries=NONE``` and ``` --disable-rpath-install``` according to https://wiki.samba.org/index.php/Waf#controlling_rpath

Reply:
Makes sense, but this should probably go in another PR.